tests/Polkadot/SS58AddressTests.cpp

```diff
@@ -0,0 +1,113 @@
+// Copyright © 2017-2022 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Polkadot/SS58Address.h"
+#include "HexCoding.h"
+#include "PublicKey.h"
+#include "../interface/TWTestUtilities.h"
+
+#include <gtest/gtest.h>
+#include <map>
+
+using namespace TW;
+
+TEST(SS58Address, IsValid) {
+    EXPECT_TRUE(SS58Address::isValid("15KRsCq9LLNmCxNFhGk55s5bEyazKefunDxUH24GFZwsTxyu", 0));
+    
+    EXPECT_TRUE(SS58Address::isValid("VNpyemgLHFKxYhWKzSKM5SoGYyAqtBSdyRaQ3DcNEsT9jpVnx", 110));
```

Comment:
Are those networks really existed? where can we cross verify our address test cases?

Reply:
Good point, I did not find official test to cross-check

Reply:
Cross-checked with Substrate implementation from https://github.com/paritytech/substrate/blob/master/primitives/core/src/crypto.rs